runtime/lua/vim/_meta/vimfn.lua

```diff
@@ -8049,7 +8049,7 @@ function vim.fn.shellescape(string, special) end
 --- will be assumed.
 ---
 --- @param col? integer
---- @return any
+--- @return integer
```

Comment:
Sorry for the unrelated change. When using this function I noticed the missing type annotations and decided to piggy-back the fix here.

Reply:
Stale comment. I ended up doing this in a separate PR.